include/swift/AST/DiagnosticsSema.def

```diff
@@ -3148,6 +3148,9 @@ ERROR(assignment_bang_has_immutable_subcomponent,none,
 NOTE(change_to_mutating,none,
      "mark %select{method|accessor}0 'mutating' to make 'self' mutable",
      (bool))
+NOTE(masked_instance_variable,none,
+     "add explicit 'self.' to make %0 an instance variable which is mutable",
```

Comment:
How about - `reference to mutable property requires explicit 'self.'`?

Reply:
I think `add explicit 'self.' to refer to mutable property of %0` is a good message for a note!